onnxscript/converter.py

```diff
@@ -844,21 +844,16 @@ def translate_opset_expr(self, node) -> values.Opset:
 
     def translate_callee_expr(self, node) -> values.Op:
         """Return an Op"""
```

Comment:
Worth fixing this check?

Reply:
Done (I hope). I added the `NoReturn` annotation to the `fail` methods.

Reply:
Thanks!

Reply:
Looks like CodeQL isn't smart enough. that's ok